src/app/state/providers/tags-state-actions.ts

```diff
@@ -41,12 +41,50 @@ export class TagsStateActions extends StateActionBranch<AppState> {
         this.tags.loadCount--;
     }
 
-    createTagFamilySuccess(tagFamily: TagFamilyResponse) {
+    clearAll() {
+        this.entities = mergeEntityState(this.entities, {
```

Comment:
Add TODO: this can be removed once there are no other instances of components getting a tags list by directly querying the `entities.tag` or `entities.tagFamily` state branch.